File: CS50-Psets/Week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    //getting user input for height
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    //for loop for printing column
    for (int i = 0; i < n; i++)
    {

        //prints right aligned pyramid
        for (int j = 0; j < n; j++)
        {
            if (i + j < n - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }

        //prints blank space in the middle
        printf("  ");

        //prints left aligned pyramid
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: CS50-Psets/Week6/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

while True:
    # getting user input
    height = get_int("Height:")
    # limit the height
    if height > 0 and height < 9:
        for i in range(height):
            for j in range(height):

                # print column
                if (i + j < height - 1):
                    print(" ", end="")
                else:
                    print("#", end="")
            print("")
        # if successful break out of the loop
        break